sdk-setup/src/sdk-chroot

```diff
@@ -1,19 +1,14 @@
 #!/bin/bash
-# mer-sdk-chroot
-
-# TODO
-#
-# Support a Mer clean setup (ie no .oscrc)
-# Support multiple shells (ie split setup/entry)
+# sdk-chroot
 
 usage()
 {
     cat <<EOF
     usage: $0 [-u <user>] [-m <all|none|root|home>] [-r <SDK root path>] [<command> <args> ..]
            $0 -h
 
-       This is the Mer chroot SDK.
-       For information see https://sailfishos.org/wiki/Platform_SDK
+       This is the Sailfish OS chroot SDK.
```

Comment:
maybe

```suggestion
       This is the minimal, chroot-based Sailfish SDK.
```

Reply:
Also don't remove the link to the wiki.

Reply:
> Also don't remove the link to the wiki.

I didn't remove the link. I replaced it with link to the new documentation site.